### ISSUE TITLE
Does it work with null value ?

### ISSUE THREAD

**Issue by Quentintin** [apurvapatel141092/sort-json-array#1]
Hello,

I currently trying to sort an array of JSON object on my AngularJS Website for a datatable.

I'm looking for a function able to sort my array by key. 
I made my own function but I got stuck because Javascript don't handle comparison between a String and a null value.

I explain : here my data : 

```
[ 
{ name : "john", date : "2017-01-08" }, 
{ name : "michel", date : null },
{ name : "toto", date : "2017-01-10"  },
{ name : "foo", date : null } 
]
```

Does your function is able to handle null value to sort my array with null value at the end ?

I made my own function, but my method seems to be too heavy, and operation take about 2 sec with array of 10 000 objects.

**Comment by apurvapatel141092** [comment]
Hi,

I have published a new release 0.1.7 which will handle null values as well. Please update your sort-json-array package. 

```
npm update sort-json-array -g
```

Usage:

``` js
var sortJsonArray = require('sort-json-array');
var data = [ 
  { name : "john", date : "2017-01-08" }, 
  { name : "michel", date : null },
  { name : "toto", date : "2017-01-10"  },
  { name : "foo", date : null } 
];
sortJsonArray(data,"asc");
```

Data will be sorted in ascending order as following:

```
[ 
  { name: 'michel', date: null },
  { name: 'foo', date: null },
  { name: 'john', date: '2017-01-08' },
  { name: 'toto', date: '2017-01-10' } 
]
```

Thanks

**Comment by Quentintin** [comment]
Ok thank you, it's perfect ! :) 

Just one last thing, you do not have to consider it.
In my case, I have a string with space at the beginning (mistake from user in edit field) like : " my name".

So when I sort, this value appears in first because space character is first. So it could be interesting to implement **trim()** function to delete useless spaces.  It should not be an obligation but an option for user.

Goodbye.
